node/src/service.rs

```diff
@@ -40,70 +38,37 @@ use sp_keystore::KeystorePtr;
 use sp_runtime::{app_crypto::AppCrypto, traits::BlakeTwo256};
 use substrate_prometheus_endpoint::Registry;
 
-/// Devnet Native executor type.
-pub struct DevnetRuntimeExecutor;
-
-impl NativeExecutionDispatch for DevnetRuntimeExecutor {
-	type ExtendHostFunctions = frame_benchmarking::benchmarking::HostFunctions;
-
```

Comment:
When the node is built with benchmarks, this host function is necessary -- otherwise we can't launch the chain. Please see this as an example: https://github.com/frequency-chain/frequency/pull/2104/files#diff-6c46aca47666767beb016249b3433e0f1703b0f8113ac2fd3d3514cd667e548dR60

Just as a note: we should always be careful about removing HostFunctions (in this case it's just benchmarking, so not a big problem). https://substrate.stackexchange.com/a/3162/703

Reply:
Thank you Peter, will update the PR following this. Lesson learnt!

Reply:
Updated the PR to undo the removal of benchmarking HostFunctions.